doc/guidelines/dev_process.md

```diff
@@ -0,0 +1,23 @@
+# Development process
+
+The [Backlog](https://github.com/orgs/sprinteins/projects/7/views/1) and current [Sprint Backlog](https://github.com/orgs/sprinteins/projects/7/) can be found on github, next to the [project repo](https://github.com/sprinteins/oscd-plugins).
+
+You need to be able to interact with Work Items :
+* create, read, update, and delete issues
+* add them to the project in order to make them appear in the backlog
+
+## Version control management practice
+
+We are following the [trunk-based development rules](https://trunkbaseddevelopment.com/) : one trunk (main), short life feature branches that should not live longer than a sprint. 
+
+## Sprint Backlog
+
+* When you are assigned to an issue from the Sprint Backlog, please do the following:
+	* Create a ney branch following the
```

Comment:
The rest of the sentence is missing :)

Reply:
Ah oups ! I made a proposition for a simple branch naming convention.